controllers/storagecluster/external_resources.go

```diff
@@ -369,37 +386,51 @@ func (r *StorageClusterReconciler) createExternalStorageClusterResources(instanc
 }
 
 // createExternalStorageClusterConfigMap creates configmap for external cluster
-func (r *StorageClusterReconciler) createExternalStorageClusterConfigMap(cm *corev1.ConfigMap, found *corev1.ConfigMap, objectKey types.NamespacedName) error {
+func (r *StorageClusterReconciler) createExternalStorageClusterConfigMap(cm, found *corev1.ConfigMap, objectKey types.NamespacedName) error {
```

Comment:
I prefer the original signature as it is easier to understand.

Reply:
Done